src/pynws/raw_data.py

```diff
@@ -20,11 +21,37 @@ def get_header(userid: str) -> Dict[str, str]:
     return {"accept": API_ACCEPT, "User-Agent": API_USER.format(userid)}
 
 
+def get_params(
+    limit: Optional[int] = 0,
+    start: Optional[datetime] = None,
+    units: Optional[str] = None,
+) -> Dict[str, Any]:
```

Comment:
This returns `Dict[str, str]`?

Reply:
The values of the dict can be a str for the start or units keys or an int for the limit parameter. That's why I put the Any.